src/NuGet.Clients/NuGet.SolutionRestoreManager/VulnerablePackagesInfoBar.cs

```diff
@@ -138,12 +139,14 @@ public void OnActionItemClicked(IVsInfoBarUIElement infoBarUIElement, IVsInfoBar
 
         protected InfoBarModel GetInfoBarModel()
         {
+            IEnumerable<IVsInfoBarTextSpan> textSpans = new IVsInfoBarTextSpan[]
```

Comment:
```suggestion
            var textSpans = new IVsInfoBarTextSpan[]
```

I'd let the compiler implicitly convert the type in the method call where it's used